### ISSUE TITLE
[Expressive modal]: QA: The button labelled "Lorem ipsum dolor" Is not keyboard operable 

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9427]
### Description

Page Element:
The button element in the modal dialog

Issue Description:
The button labelled "Lorem ipsum dolor" Is not keyboard operable for the screen reader users.

Expected:
Make it keyboard operable

Impact:
When interactive controls are inoperable with keyboard, User is unable to perform an action on the web page or complete a task. This defect impacts Keyboard only users like Blind screen reader users and users with motor disabilities.

### Component(s) impacted

Expressive Modal

### Browser

Firefox

### Carbon for IBM.com version

v1.39.0-rc.0

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

Web component Canary- https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components Web component Staging- https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
